docs/source/index.rst

```diff
@@ -0,0 +1,4 @@
+The AaC Specification (aac-spec) Plugin is an extension of the core AaC tool and provides specification and requirements traceability capabilities upon ingest of an AaC model. 
+Documentation for the core AaC tool can be found `here <https://arch-as-code.org/>`_"
```

Comment:
```suggestion
Documentation for the core AaC tool can be found `here <https://arch-as-code.org/>`_
```

Missed removing an end quote from copying in the comment in the initial suggestion.

Reply:
Done